### PR TITLE
Adding a my jetus.is.a.dev

### PR DESCRIPTION
**Pull request by j3tus**

### CODE REVIEW

File: domains/jetus.is.a.dev [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "A personal is a dev site.",
+  "repo": "https://github.com/Dev-Jetus/register",
```
Comment (phenax) [inline domains/jetus.is.a.dev:3]: This should be the repo hosting your github pages site

File: domains/jetus.is.a.dev [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  "owner": {
+    "username": "jetus",
+    "email": "officialjetus@gmail.com",
+    "twitter": "itzjetus"
+  },
+  "record": {
+    "CNAME": "https://github.com/dev-jetus/dev-jetus.github.io"
```
Comment (phenax) [inline domains/jetus.is.a.dev:10]: `CNAME` has to be a hostname. In this case, I think you meant `dev-jetus.github.io`

**Review by phenax — CHANGES_REQUESTED** [review]

File: domains/jetus.is.a.dev (hunk 2 above)
Comment (Dev-Jetus) [reply domains/jetus.is.a.dev:10]: Can i just leave this for a bit, and i will do this later on.

Comment (TheBotlyNoob) [reply domains/jetus.is.a.dev:10]: 🤦‍♂️ Just change the CNAME to dev-jetus.github.io then you can get the subdomain...

**Pull request closed without merging**